classes/controller/AdminController.php

```diff
@@ -408,8 +408,14 @@ class AdminControllerCore extends Controller
     /** @var string */
     protected $tabSlug;
 
-    /** @var int Auth cookie lifetime */
+    /** 
+     * @deprecated use $this->authCookieLifetime
```

Comment:
should follow a deprecation guidelines https://devdocs.prestashop.com/1.7/development/coding-standards/#deprecations
there are nothing about constants or properties so I think its enough to just change wording to:
```suggestion
     * @deprecated since version 8.0.1. Use $this->authCookieLifetime instead
```

Reply:
altough sorry for a little blocker, I would like other opinions from @PrestaShop/committers. If we do this change then property wouldn't be reached from outside (e.g. maybe some modules are using it?), then maybe it should be static?

Reply:
I think this should be a property because this way the admin can modify this value from BO. But if that is a constant, they only can do this in the file.

Reply:
yeah but what I mean you could previously use the constant value in any other php class like so:
`AdminController::AUTH_COOKIE_LIFETIME`
Now if you make it non static property, you will only be able to use it after the class is initiallized, so will not be able to call smth like AdminController::$authCookieLifetime, so I thought maybe it could be a static property.
But on second thought, AdminController will be initialized anyway in any context anyway, so anyone can reach it through $context->controller->$authCookieLifetime, so probably this won't be a problem :smile: